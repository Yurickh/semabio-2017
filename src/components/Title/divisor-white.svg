<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 101 4" style="enable-background:new 0 0 101 4;" xml:space="preserve">
<style type="text/css">
	.st0{filter:url(#Adobe_OpacityMaskFilter);}
	.st1{fill:#FFFFFF;}
	.st2{mask:url(#mask-2);fill:#FFFFFF;}
</style>
<title>Page 1</title>
<desc>Created with Sketch.</desc>
<g id="Page-1_1_">
	<g id="BetterDesk" transform="translate(-659.000000, -860.000000)">
		<g id="Page-1" transform="translate(659.000000, 860.000000)">
			<g id="Clip-2">
			</g>
			<defs>
				<filter id="Adobe_OpacityMaskFilter" filterUnits="userSpaceOnUse" x="0" y="0" width="100.7" height="3.8">
					<feColorMatrix  type="matrix" values="1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 1 0"/>
				</filter>
			</defs>
			<mask maskUnits="userSpaceOnUse" x="0" y="0" width="100.7" height="3.8" id="mask-2">
				<g class="st0">
					<polygon id="path-1" class="st1" points="0,3.8 0,0 100.7,0 100.7,3.8 					"/>
				</g>
			</mask>
			<path id="Fill-1" class="st2" d="M10.7,3.5c1.8,0.1,3.7,0.2,5.5,0.2c2.6,0,5.3,0,7.9-0.1c2.3,0,4.6-0.1,6.8-0.1
				c1.8,0,3.6-0.1,5.5-0.1c2.1,0,4.2-0.1,6.3-0.1c4.8-0.1,9.7-0.3,14.5-0.4c1.9,0,3.8-0.1,5.8-0.1c1.4,0,2.8,0,4.2,0
				c0.6,0,1.3,0,1.9,0c1.5,0,3.1,0,4.6,0.1c0.7,0,1.3,0,2,0c2.2,0.1,4.4,0.2,6.7,0.3c2.4,0.1,4.9,0.2,7.3,0.3
				c2.1,0.1,4.2,0.1,6.4,0.1c1.6,0,3.1-0.1,4.7-0.2c-1-0.2-2-0.3-3.1-0.4c0,0,0.1,0,0.1,0c-2.2-0.3-4.5-0.6-6.8-0.8
				c-1.8-0.2-3.5-0.3-5.3-0.5c0.7,0,1.3-0.1,2-0.1c0.9,0,1.9-0.1,2.8-0.1c1.2-0.1,2.4-0.1,3.6-0.2c2.1-0.1,4.3-0.3,6.4-0.5
				c-2.2-0.2-4.4-0.2-6.5-0.3c-2.3,0-4.6,0-6.9,0c-2.2,0-4.4,0.1-6.6,0.1c-2,0.1-3.9,0.1-5.9,0.1c-1.6,0-3.2,0.1-4.8,0.1
				c-0.8,0-1.6,0-2.3,0c-2.6,0-5.2,0-7.8,0c-0.7,0-1.3,0-2,0c-1.1,0-2.3,0-3.4,0c-1.5,0-3.1,0-4.6,0c-0.6,0-1.1,0-1.7,0
				c-2.7-0.1-5.5-0.1-8.2-0.2c-2.2-0.1-4.3-0.1-6.5-0.2C29.1,0.4,25,0.2,20.9,0.1C18.8,0.1,16.6,0,14.5,0c-0.6,0-1.1,0-1.7,0
				c-1,0-2,0-3,0c-0.8,0-1.5,0-2.3,0C5.1,0.1,2.5,0.1,0,0.1c0.7,0.1,1.3,0.2,2,0.3c0.7,0.1,1.4,0.2,2,0.3C5.3,0.8,6.5,1,7.8,1.1
				c0.2,0,0.5,0,0.7,0.1c0.7,0.1,1.3,0.1,2,0.1c1.5,0.1,2.9,0.2,4.4,0.3c0.8,0,1.7,0.1,2.5,0.1C13.3,1.8,9.3,2.1,5.2,2.5
				C4,2.6,2.9,2.7,1.7,2.8C4.7,3.1,7.7,3.4,10.7,3.5z"/>
		</g>
	</g>
</g>
</svg>
