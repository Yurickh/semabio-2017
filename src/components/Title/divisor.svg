<?xml version="1.0" encoding="UTF-8"?>
<svg width="101px" height="4px" viewBox="0 0 101 4" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>Page 1</title>
    <desc>Created with Sketch.</desc>
    <defs>
        <polygon id="path-1" points="0 3.7563 0 -9.99999881e-05 100.711016 -9.99999881e-05 100.711016 3.7563"></polygon>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-659.000000, -860.000000)">
            <g id="Page-1" transform="translate(659.000000, 860.000000)">
                <mask id="mask-2" fill="white">
                    <use xlink:href="#path-1"></use>
                </mask>
                <g id="Clip-2"></g>
                <path d="M10.7488322,3.53826 C12.5775506,3.63506 14.4070935,3.71026 16.2382853,3.74026 C18.8758282,3.78386 21.511722,3.73066 24.1488527,3.67946 C26.4244445,3.63506 28.7012731,3.60786 30.9772771,3.57226 C32.8006363,3.54466 34.6235833,3.51066 36.446118,3.47626 C38.5473302,3.43666 40.6493669,3.41226 42.7493424,3.35306 C47.5792037,3.21586 52.4086527,3.08346 57.2393384,2.96546 C59.1591629,2.91786 61.0781629,2.89386 62.9992241,2.87306 C64.3946731,2.85786 65.790122,2.84306 67.1859833,2.82826 C67.816718,2.82186 68.4466282,2.83466 69.0777751,2.84306 C70.6162731,2.86466 72.1551833,2.88586 73.6940935,2.90746 C74.3635792,2.91666 75.0326527,2.91986 75.7013139,2.95226 C77.9257873,3.05986 80.1510853,3.14586 82.3739098,3.27866 C84.8040935,3.42386 87.2408731,3.52586 89.6751792,3.60066 C91.794118,3.66546 93.9155302,3.71426 96.036118,3.69786 C97.5948159,3.68546 99.1601098,3.64666 100.709326,3.45946 C99.6869588,3.30866 98.6670649,3.14786 97.6426363,3.01586 C97.672318,3.01946 97.7019996,3.02386 97.7316812,3.02746 C95.4820608,2.71626 93.2196608,2.46826 90.9576731,2.25426 C89.1986241,2.08786 87.4387506,1.93266 85.6768159,1.79626 C86.3322853,1.76906 86.9873424,1.74026 87.6419873,1.71026 C88.5707751,1.66666 89.4983261,1.61306 90.4271139,1.57066 C91.6119057,1.51706 92.7958731,1.44026 93.9806649,1.38266 C96.1272241,1.27906 98.2713098,1.13226 100.410449,0.93026 C98.2428649,0.71626 96.0571424,0.69466 93.8809016,0.65466 C91.5710935,0.61266 89.2588118,0.60706 86.9481792,0.62106 C84.7488527,0.63426 82.5503506,0.67306 80.3518486,0.73306 C78.3842037,0.78746 76.4177955,0.84026 74.4505629,0.87826 C72.8580608,0.90906 71.265971,0.93986 69.6742935,0.97066 C68.8984486,0.98506 68.1213669,1.01146 67.3459343,1.01466 C64.7421955,1.02506 62.138869,1.03666 59.5351302,1.04706 C58.8763629,1.05026 58.216771,1.06066 57.5575914,1.05426 C56.4099016,1.04226 55.2613873,1.03066 54.1136976,1.01946 C52.580971,1.00386 51.0478322,0.98866 49.515518,0.97346 C48.9589873,0.96746 48.4020445,0.96386 47.8459261,0.95066 C45.1127424,0.88626 42.3791465,0.82186 39.6455506,0.75746 C37.4754935,0.70666 35.3058486,0.61386 33.1362037,0.54026 C29.0656976,0.40186 24.9951914,0.24866 20.9234486,0.13106 C18.7698812,0.06906 16.6175506,0.02386 14.463571,0.00626000002 C13.9086894,0.00226000002 13.35422,-0.000139999983 12.7993384,-0.000139999983 C11.8153098,-0.000139999983 10.8316935,0.00666000002 9.8476649,0.01866 C9.09490571,0.02746 8.34173429,0.05306 7.58815061,0.05226 C5.05944041,0.05026 2.5307302,0.11226 0.0016077551,0.12506 C0.666146531,0.23466 1.33027306,0.33946 1.9981098,0.42466 C2.67501592,0.51106 3.35233429,0.59466 4.02965265,0.67546 C5.2824649,0.82546 6.53568939,0.96506 7.78973837,1.09706 C8.02183224,1.12186 8.2555751,1.13506 8.48808122,1.15306 C9.1563302,1.20426 9.82457918,1.25306 10.4936527,1.29906 C11.9661914,1.40026 13.4387302,1.48746 14.9125057,1.55946 C15.7423547,1.59986 16.5717914,1.62706 17.4020527,1.64946 C13.3336078,1.78946 9.26887306,2.07506 5.21980367,2.47426 C4.04160776,2.59066 2.86341184,2.70746 1.68727714,2.83626 C4.70202408,3.14546 7.72171796,3.37786 10.7488322,3.53826 Z" id="Fill-1" fill="#2F5035" mask="url(#mask-2)"></path>
            </g>
        </g>
    </g>
</svg>