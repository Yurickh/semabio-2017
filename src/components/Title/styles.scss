@import 'ambience/typography';
@import 'ambience/responsive';

$simple-title-color-darkslategrey: #2F5035;

.simple-title {
  font-size: 32px;

  display: flex;
  align-items: center;
  flex-direction: column;

  text-transform: uppercase;

  color: $simple-title-color-darkslategrey;

  & > .divisor {
    width: 100px;
    height: 4px;
    margin-top: 12px;
    margin-bottom: 40px;
  }
}
