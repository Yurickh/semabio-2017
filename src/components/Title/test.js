import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import Title from '../Title'

describe('Title', () => {
  it('should render correctly', () => {
    const button = shallow(<Title />)
    expect(toJSON(button)).toMatchSnapshot()
  })
})
