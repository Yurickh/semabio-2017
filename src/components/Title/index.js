import React, { Component } from 'react'
import PropTypes from 'prop-types'

import './styles.css'

class Title extends Component {
  render() {
    return (
      <div className={`simple-title -${this.props.color}`}>
        { this.props.children }
        <div className="divisor"></div>
      </div>
    )
  }
}
Title.propTypes = {
  color: PropTypes.oneOf(['white'])
}

Title.defaultProps = {
  color: ''
}

export default Title
