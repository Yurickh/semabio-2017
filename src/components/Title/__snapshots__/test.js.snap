// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<div
  className="simple-title  "
>
  <div
    className="divisor"
  />
</div>
`;
